src/providers/recordedit.tsx

```diff
@@ -575,7 +575,8 @@ export default function RecordeditProvider({
 
           // NOTE: This check was being done in angularJS since the modal was closed with a message if the user aborted uploading
           //   - in ReactJS it's handling the case when the modal is closed and no "exception" is returned (undefined)
-          if (typeof exception !== 'string') {
+          //   - when we abort the upload, we're calling onCancel without any parameters
+          if (isObjectAndNotNull(exception)) {
```

Comment:
This is better than what I left this case as. I wasn't sure how to handle this before, it makes more sense that we want to check for object and not null since now we know a string will never be returned.